rules/windows/persistence_via_telemetrycontroller_scheduledtask_hijack.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/08/17"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the successful hijack of Microsoft Compatibility Appraiser scheduled task to persist rogue program with system
+Integrity Level.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Persistence via TelemetryController Scheduled Task Hijack"
+references = [
+    "https://www.trustedsec.com/blog/abusing-windows-telemetry-for-persistence/?utm_content=131234033&utm_medium=social&utm_source=twitter&hss_channel=tw-403811306",
+]
+risk_score = 73
+rule_id = "68921d85-d0dc-48b3-865f-43291ca2c4f2"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type:(start or process_started) and
```

for process events, I would be explicit be using "start" because there is also an event.type of info which may result in unrelated or duplicative alerts. process_started is for backwards compat for pre 7.8 beats